docs/source/gbd2017_models/risk_attributable_causes/chronic_kidney_disease/cause_model_ckd.svg

```diff
@@ -1 +1 @@
)" xlink:href="#ab"/><use transform="matrix(0.05,0,0,0.05,10.950000000000001,0)" xlink:href="#C"/><use transform="matrix(0.05,0,0,0.05,20.950000000000003,0)" xlink:href="#y"/><use transform="matrix(0.05,0,0,0.05,24.900000000000002,0)" xlink:href="#N"/><use transform="matrix(0.05,0,0,0.05,28.85,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,38.85,0)" xlink:href="#H"/><use transform="matrix(0.05,0,0,0.05,44.800000000000004,0)" xlink:href="#E"/></g><path d="M240 0l2-218c-23 76-54 145-80 218h-23L58-218 59 0H30v-248h44l77 211c21-75 51-140 76-211h43V0h-30" id="ac"/><g id="t"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#ac"/><use transform="matrix(0.05,0,0,0.05,14.950000000000001,0)" xlink:href="#G"/><use transform="matrix(0.05,0,0,0.05,24.950000000000003,0)" xlink:href="#aa"/><use transform="matrix(0.05,0,0,0.05,34.95,0)" xlink:href="#E"/><use transform="matrix(0.05,0,0,0.05,44.95,0)" xlink:href="#H"/><use transform="matrix(0.05,0,0,0.05,50.900000000000006,0)" xlink:href="#C"/><use transform="matrix(0.05,0,0,0.05,60.900000000000006,0)" xlink:href="#D"/><use transform="matrix(0.05,0,0,0.05,65.9,0)" xlink:href="#E"/></g><path d="M108 0H70L1-190h34L89-25l56-165h34" id="ad"/><g id="u"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#K"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#E"/><use transform="matrix(0.05,0,0,0.05,22,0)" xlink:href="#ad"/><use transform="matrix(0.05,0,0,0.05,31,0)" xlink:href="#E"/><use transform="matrix(0.05,0,0,0.05,41,0)" xlink:href="#H"/><use transform="matrix(0.05,0,0,0.05,46.95,0)" xlink:href="#E"/></g><path d="M194 0L95-120 63-95V0H30v-248h33v124l119-124h40L117-140 236 0h-42" id="ae"/><g id="v"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#ae"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#y"/><use transform="matrix(0.05,0,0,0.05,15.949999999999998,0)" xlink:href="#aa"/><use transform="matrix(0.05,0,0,0.05,25.950000000000003,0)" xlink:href="#Z"/><use transform="matrix(0.05,0,0,0.05,35.95,0)" xlink:href="#E"/><use transform="matrix(0.05,0,0,0.05,45.95,0)" xlink:href="#I"/></g><path d="M127-220V0H93v-220H8v-28h204v28h-85" id="af"/><path d="M115-194c55 1 70 41 70 98S169 2 115 4C84 4 66-9 55-30l1 105H24l-1-265h31l2 30c10-21 28-34 59-34zm-8 174c40 0 45-34 45-75s-6-73-45-74c-42 0-51 32-51 76 0 43 10 73 51 73" id="ag"/><g id="w"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#af"/><use transform="matrix(0.05,0,0,0.05,10.3,0)" xlink:href="#H"/><use transform="matrix(0.05,0,0,0.05,16.25,0)" xlink:href="#C"/><use transform="matrix(0.05,0,0,0.05,26.25,0)" xlink:href="#Z"/><use transform="matrix(0.05,0,0,0.05,36.25,0)" xlink:href="#z"/><use transform="matrix(0.05,0,0,0.05,45.25,0)" xlink:href="#ag"/><use transform="matrix(0.05,0,0,0.05,55.25,0)" xlink:href="#N"/><use transform="matrix(0.05,0,0,0.05,59.2,0)" xlink:href="#C"/><use transform="matrix(0.05,0,0,0.05,69.2,0)" xlink:href="#Z"/><use transform="matrix(0.05,0,0,0.05,79.2,0)" xlink:href="#D"/></g></defs></g></svg>
```

Comment:
I appreciate the new model diagram.  
We are not including kidney transplant in the simulation model, however.  Is this figure supposed to represent what we are actually planning on using, or a more general model?

Reply:
It should be a generalized model that we are using. Should I update the cause model diagram to remove 'kidney transplant'?

Reply:
Yes.  The diagram should correspond to the simulation model.  You can include other diagrams for comparison if you'd like.

Reply:
I updated the cause model diagram to remove 'kidney transplant'. I do not think it is complete yet, but is on the right track. @davidbernotas what do you recommend the cause model diagram to look like to connect between the cause, sequelae, and risk factor categories?